docs/content/preview/drivers-orms/java/ebean.md

```diff
@@ -0,0 +1,244 @@
+
+1. To use this directory as a default Ebean package of classes, add the following code at the end of the `conf/application.conf` file:
+
+    ```conf
+    ebean.default="models.*"
+    ```
+
+1. Create a `Employee.java` file in `app/models/`. This is the class definition or information for the employee. Add the following code to the file:
+
+    ```java
+    package models;
+
+    import io.ebean.Finder;
+    import io.ebean.Model;
+    import javax.persistence.*;
+    import javax.validation.constraints.NotBlank;
+
+    @Entity
+    @Table(name = "employee")
+    public class Employee extends Model{
+
+      @Id
+      @GeneratedValue(strategy = GenerationType.IDENTITY)
+      @Column(columnDefinition = "serial")
+      public Long empId;
+
+      @NotBlank
+      public String firstName;
+
+      @NotBlank
+      public String lastName;
+
+      @Column(name = "emp_email")
+      public String email;
+
+      public static final Finder<Long, Employee> find = new Finder<>(Employee.class);
+
+      @Override
+      public String toString(){
+          return "{'empId' = '"+empId+"', firstName ='"+firstName+"', 'lastName'      ='"+lastName+"', 'email' ='"+email+"' }";
+
+      }
+    }
+    ```
+
+1. Create an `EmployeeController.java` file under `app/controllers/` directory. This file controls the flow of employees data. It consists of methods for all API calls, including adding an employee, and retrieving employee information. Using the annotation `@Transactional` over the APIs gives the feature of automatically managing the transaction in that API. Add the following code to the file:
+
+    ```java
+    package controllers;
+    import models.Employee;
+    import javax.persistence.*;
+    import play.libs.Json;
+    import play.db.ebean.Transactional;
+    import play.mvc.*;
+    import java.util.ArrayList;
+    import java.util.List;
+    public class EmployeeController extends Controller{
+      @Transactional
+      public Result AddEmployee(Http.Request request){
+          Employee employee=Json.fromJson(request.body().asJson(),Employee.class);
+          employee.save();
+          return ok(Json.toJson(employee.toString()));
+      }
+      public Result GetAllEmployees(){
+          List <Employee> employees = Employee.find.all();
+          List<String> employeesList = new ArrayList<String>();
+          for(int index=0;index<employees.size();index++){
+              employeesList.add(employees.get(index).toString());
+          }
+          return ok(Json.toJson(employeesList));
+      }
+    }
+    ```
+
+1. Add the GET and POST API Request for the `/employees` endpoint to the `conf/routes` file. This will define the necessary method for receiving the request:
+
+    ```conf
+    GET      /employees            controllers.EmployeeController.GetAllEmployees
+    POST     /employees            controllers.EmployeeController.AddEmployee(request: Request)
+    ```
+
+### Compile and run the project
+
```

Comment:
And some small intro like "To run the application and insert a new row, execute the following steps:" or somesuch.